addon/mixins/oidc-application-route-mixin.js

```diff
@@ -2,6 +2,22 @@ import ApplicationRouteMixin from "ember-simple-auth/mixins/application-route-mi
 import Mixin from "@ember/object/mixin";
 
 export default Mixin.create(ApplicationRouteMixin, {
+  sessionAuthenticated() {
+    /**
+     * If there is a stored `continueTransition` URL in the localstorage
+     * transition to it otherwise call the parent/super to invoke the normal
+     * behavior of the `sessionAuthenticated` hook.
+     */
```

Comment:
Please do a real docstring above the method

Reply:
But the comment describes what happens in the function and not what the function actually does. So I don't think it would be fitting as a method docstring. Or what was your thought behind this?

Reply:
IMHO that's exactly what docstrings should be used for.. What do you think @czosel?

Reply:
Changed it after our discussion ;)